es6/es6.tag

```diff
@@ -0,0 +1,10 @@
+<es-6>
+
+  <p>{ message }</p>
+
+  <script type="es6">
+    const name = 'Riot'
+    this.message = `Hello ${name}`
```

Comment:
I just thought the point was to show how to get Riot using es6, rather than how to write es6. 

I can add an example method if you'd like?